exchange/bidder.go

```diff
@@ -328,6 +337,21 @@ func (bidder *bidderAdapter) doRequest(ctx context.Context, req *adapters.Reques
 	}
 }
 
+func (bidder *bidderAdapter) doTimeoutNotification(timeoutBidder adapters.TimeoutBidder, req *adapters.RequestData) {
+	ctx, cancel := context.WithTimeout(context.Background(), 200*time.Millisecond)
+	defer cancel()
```

Comment:
Why do we need to defer the cancellation?

Reply:
The answer: memory leaks
https://stackoverflow.com/questions/44393995/what-happens-if-i-dont-cancel-a-context